EnglishFilter/sections/specific.txt

```diff
@@ -184,7 +184,7 @@ ww.9to5google.com,ww.9to5mac.com,ww.9to5toys.com,ww.dronedj.com,ww.electrek.co##
 ! ##.banner
 photoeditor.com,9c717baaf805a8436afd7912039e826c.link,uflash.tv,boxlink.us,shortbitsfree.net,hellomagazine.com,adsme.cc,clickurls.site,planetanime.co,rocketfiles.com,pornwhite.com,aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.com,govtsmartjob.com,cutearn.ca,dfporn.net,cll.press,hellporno.net,youramateurporn.com,getlink.pw,embedtvseries.com,keygames.com,uclick.in,flv2mp3.by,doodhwali.com,smutindia.com,dvdcover.com,xxxadultphoto.com,gabicoins.com,qpdownload.com,windows10portal.com,2conv.com,javcloud.com,underhentai.net,cutwin.com,hqasiananal.com,bradsknutson.com,flv2mp3.org,subscene.com,yourlink.in,iplayer.pw,qpornx.com,sofifa.com,jawcloud.co,indiansgetfucked.com,medi-bayreuth.de,oxforddictionaries.com,tube18.sex##.banner
 ! ##.ad
-nejm.org,ertyu.org,howchoo.com,dvdsreleasedates.com,unicode-search.net,icefilms.info,saavn.com,msn.com,www-technologyreview-com.cdn.ampproject.org,greasyfork.org,ranker.com,curiouscat.me,pornhd.com,history.com,investopedia.com,mangafox.la,sexhoundlinks.com,ibc.org,dcode.fr,tempostorm.com,defensesystems.com,whatsmydns.net,newatlas.com,gtplanet.net,livescore.com,xcafe.com,ancient.eu,gradesaver.com,nytimes.com,nytimes3xbfgragh.onion,thurrott.com,mtlblog.com,accuweather.com,narcity.com,mywebtimes.com,cellz.io,forvo.com,apornotube.net,instadp.com,stream365.xyz,bab.la,elophant.com,ibrodtv.net,ipaddress.com,mobafire.com,movieweb.com,ndtv.com,owfire.com,samistream.com,trafficland.com##.ad
+instadp.com,nejm.org,ertyu.org,howchoo.com,dvdsreleasedates.com,unicode-search.net,icefilms.info,saavn.com,msn.com,www-technologyreview-com.cdn.ampproject.org,greasyfork.org,ranker.com,curiouscat.me,pornhd.com,history.com,investopedia.com,mangafox.la,sexhoundlinks.com,ibc.org,dcode.fr,tempostorm.com,defensesystems.com,whatsmydns.net,newatlas.com,gtplanet.net,livescore.com,xcafe.com,ancient.eu,gradesaver.com,nytimes.com,nytimes3xbfgragh.onion,thurrott.com,mtlblog.com,accuweather.com,narcity.com,mywebtimes.com,cellz.io,forvo.com,apornotube.net,instadp.com,stream365.xyz,bab.la,elophant.com,ibrodtv.net,ipaddress.com,mobafire.com,movieweb.com,ndtv.com,owfire.com,samistream.com,trafficland.com##.ad
```

Comment:
This domain already existed here before you made the fix. Remove duplicate and search in optimized filters for iOS - maybe this common rule doesn't exist and it needs to add a hint `NOT_OPTIMIZED`.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/83020270-e93a2e00-a030-11ea-9566-0f7df57552b0.png)
</details><br/>

Reply:
This rule exist in optimized filter, I'll check it on iOS again tomorrow
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/83022768-b003bd00-a034-11ea-9227-f6904265d722.png">
</details>